buildSrc/src/main/java/fmlbuild/InstallProductionServerTask.java

```diff
@@ -92,7 +92,7 @@ public void install() throws Exception {
         });
 
         // We need to know the name of the main class to split the arg-file into JVM and program arguments
-        var mainClass = getMainClass();
```

Comment:
This only worked when Client+Server had the same main class and hasn't worked since the startup refactor.